grails-app/assets/javascripts/meriplan.js

```diff
@@ -329,6 +329,8 @@ function MERIPlan(project, projectService, config) {
     self.protectedNaturalAssests = ['Natural/Cultural assets managed', 'Threatened Species', 'Threatened Ecological Communities',
         'Migratory Species', 'Ramsar Wetland', 'World Heritage area', 'Community awareness/participation in NRM', 'Indigenous Cultural Values',
         'Indigenous Ecological Knowledge', 'Remnant Vegetation', 'Aquatic and Coastal systems including wetlands', 'Not Applicable'];
+    self.controls = ['Yes', 'No'];
+    self.methodType = ['Natural', 'Chemical', 'Disease Virus', 'Mechanical', 'Genetics', 'Other with comment field'];
```

Comment:
Move this methodType drop down into the configuration JSON I think as it's likely we'll be asked to change it.  Or maybe leave it here but allow it to be overwritten?

Reply:
I've pushed the changes for this